cli/tsc/00_typescript.js

```diff
@@ -141247,53 +141246,18 @@ function isImportable(program, fromFile, toFile, toModule, preferences, packageJ
-  }
-  const getCanonicalFileName = hostGetCanonicalFileName(moduleSpecifierResolutionHost);
-  const globalTypingsCache = (_a = moduleSpecifierResolutionHost.getGlobalTypingsCacheLocation) == null ? void 0 : _a.call(moduleSpecifierResolutionHost);
-  const hasImportablePath = !!ts_moduleSpecifiers_exports.forEachFileNameOfModule(
-    fromFile.fileName,
-    toFile.fileName,
-    moduleSpecifierResolutionHost,
-    /*preferSymlinks*/
-    false,
-    (toPath3) => {
-      const file = program.getSourceFile(toPath3);
-      return (file === toFile || !file) && isImportablePath(
-        fromFile.fileName,
-        toPath3,
-        getCanonicalFileName,
-        globalTypingsCache,
-        moduleSpecifierResolutionHost
-      );
-    }
-  );
-  if (packageJsonFilter) {
-    const importInfo = hasImportablePath ? packageJsonFilter.getSourceFileInfo(toFile, moduleSpecifierResolutionHost) : void 0;
-    moduleSpecifierCache == null ? void 0 : moduleSpecifierCache.setBlockedByPackageJsonDependencies(fromFile.path, toFile.path, preferences, {}, importInfo == null ? void 0 : importInfo.packageName, !(importInfo == null ? void 0 : importInfo.importable));
-    return !!(importInfo == null ? void 0 : importInfo.importable) || hasImportablePath && !!(importInfo == null ? void 0 : importInfo.packageName) && fileContainsPackageImport(fromFile, importInfo.packageName);
-  }
-  return hasImportablePath;
+  return true;
 }
 function fileContainsPackageImport(sourceFile, packageName) {
   return sourceFile.imports && sourceFile.imports.some((i) => i.text === packageName || i.text.startsWith(packageName + "/"));
 }
-function isImportablePath(fromPath, toPath3, getCanonicalFileName, globalCachePath, host) {
-  const toNodeModules = forEachAncestorDirectoryStoppingAtGlobalCache(
-    host,
-    toPath3,
-    (ancestor) => getBaseFileName(ancestor) === "node_modules" ? ancestor : void 0
-  );
-  const toNodeModulesParent = toNodeModules && getDirectoryPath(getCanonicalFileName(toNodeModules));
-  return toNodeModulesParent === void 0 || startsWith(getCanonicalFileName(fromPath), toNodeModulesParent) || !!globalCachePath && startsWith(getCanonicalFileName(globalCachePath), toNodeModulesParent);
-}
-function forEachExternalModuleToImportFrom(program, host, preferences, useAutoImportProvider, cb) {
+function forEachExternalModuleToImportFrom(program, host, preferences, useAutoImportProvider, referrerFile, cb) {
   var _a, _b;
   const useCaseSensitiveFileNames2 = hostUsesCaseSensitiveFileNames(host);
   const excludePatterns = preferences.autoImportFileExcludePatterns && getIsExcludedPatterns(preferences, useCaseSensitiveFileNames2);
-  forEachExternalModule(program.getTypeChecker(), program.getSourceFiles(), excludePatterns, host, (module2, file) => cb(
+  const importableFilePaths = host.getExportImportablePathsFromModule(referrerFile.path);
+  const files = importableFilePaths.map((path) => program.getSourceFile(path)).filter((s) => s != null);
```

Comment:
Investigated why the tests are now failing and it's because they're not in the program anymore.